cmd/podman/images/list.go

```diff
@@ -128,10 +128,30 @@ func images(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	repoTags := []string{}
+	if len(args) > 0 && !listOptions.All {
+		for _, img := range summaries {
+			for _, repoTag := range img.RepoTags {
+				if strings.Contains(repoTag, args[0]) {
```

Comment:
is `strings.Contains` the expected comparison to perform here?

e.g., if I build two images like:

```
$ podman build -t exam . 
podman build -t example . 
```

then I get:

```
$ bin/podman images exam
REPOSITORY         TAG         IMAGE ID      CREATED      SIZE
localhost/exam     latest      ca20047de3b8  3 hours ago  165 MB
localhost/example  latest      ca20047de3b8  3 hours ago  165 MB
```